functions_framework_builder/pubspec.yaml

```diff
@@ -7,7 +7,7 @@ environment:
   sdk: ">=2.12.0 <3.0.0"
 
 dependencies:
-  analyzer: ^1.0.0
+  analyzer: ^1.2.0
```

Comment:
revert this change

Reply:
@kevmoo Can you explain why?

Reply:
It's a bit of a wash. Since we're only testing w/ latest we should likely pin latest.

...there is a problem where someone using our code has explicitly excluded `analyzer v1.2` because they are avoiding a change. In which case we'll have dependency lock.

I generally only bump this if I know I'm using a new feature. 🤷